byterun/domain.c

```diff
@@ -737,6 +738,80 @@ void caml_sample_gc_stats(struct gc_stats* buf)
   }
 }
 
+void caml_print_stats () {
+  struct gc_stats s;
+  int i;
+#ifdef COLLECT_STATS
+  struct detailed_stats ds;
+  caml_domain_state* st;
+  uint64 total;
+#endif
+
+  caml_gc_stat(Val_unit);
+  caml_sample_gc_stats(&s);
+  fprintf(stderr,"**** GC stats ****\n");
+  fprintf(stderr, "Minor words:\t\t%llu\n", s.minor_words);
```

Comment:
I did find the `ARCH_INT64_PRINTF_FORMAT` specifier. In my initial reading, I missed seeing the `u` and incorrectly assumed that it was printing signed integers. I'll leave this as `llu` for now.